### PR TITLE
simplified main.py

### PR DESCRIPTION
**Pull request by BlackSound1**
This cleans up main.py by making use of train_test_split from sklearn. Also, reads the text file differently

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -30,3 +30,43 @@ def get_label_distribution(all_labels: list) -> dict:
     labels = sorted(list(set(all_labels)))
     label_counts = {label: all_labels.count(label) for label in labels}
     return label_counts
+
+def listToString(list: list) -> str:
```
Comment (EvanMateo7) [inline src/utils.py]: rename this to `list_to_string`

File: src/utils.py [hunk 2]
```diff
@@ -30,3 +30,43 @@ def get_label_distribution(all_labels: list) -> dict:
...
+
+    Returns:
+      str made from elements of the list.
+
+      e.g. "This was a list" 
+    """
+    string = " "
+    return (string.join(list))
+
+def read_data(path: str) -> (list, list):
```
Comment (EvanMateo7) [inline src/utils.py]: Don't we already have `read_documents` above that does exactly this?

**Review by EvanMateo7 — CHANGES_REQUESTED** [review]

File: src/utils.py (hunk 2 above)
Comment (BlackSound1) [reply src/utils.py]: Oh ya, sorry. I got caught up and made my own version of what already existed.

File: src/utils.py (hunk 1 above)
Comment (BlackSound1) [reply src/utils.py]: done

**Review by EvanMateo7 — APPROVED** [review]

**Review by jayohanna — APPROVED** [review]

File: src/main.py [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
-split_point = int(0.80*len(all_docs))
+X, y = utils.read_documents('input/all_sentiment_shuffled.txt')  
 
-# Get training and evaluation data
-train_docs = all_docs[:split_point]
-train_labels = all_labels[:split_point]
-eval_docs = all_docs[split_point:]
-eval_labels = all_labels[split_point:]
+# Split point between training and evaluation
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
```
Comment (jayohanna) [inline src/main.py:10]: maybe add `shuffle=False` to train_test_split()
i'm thinking since in Task 3 we need to print the row nb, maybe we need to keep the order as is

**Review by jayohanna — CHANGES_REQUESTED** [review]

File: src/main.py (hunk 3 above)
Comment (EvanMateo7) [reply src/main.py:10]: We can get the row nb from `train_test_split` if we provide another array after X and y.
We get the lists with indices for the train and test sets.
```python
X_train, X_test, y_train, y_test, indices_train, indices_test = train_test_split(all_docs, all_labels, np.arange(len(all_docs)), test_size=0.2)

Comment (jayohanna) [reply src/main.py:10]: I wasn't aware of that, cool thanks

**Review by jayohanna — APPROVED** [review]

File: src/main.py (hunk 3 above)
Comment (jayohanna) [reply src/main.py:10]: wait, okay yea i see now lol, that makes sense
